components/SplitsDisplay.vue

```diff
@@ -135,14 +141,24 @@ export default class SplitsDisplay extends Vue {
     return asArray(this.parsedSplits.Run.AttemptHistory.Attempt).find((a) => a['@_id'] == this.currentAttemptNumber) || this.PB;
   }
 
-  get allRunAttempts(): Attempt[] {
-    return asArray(this.parsedSplits?.Run.AttemptHistory.Attempt);
+  get runAttempts(): Attempt[] {
+    if (this.filterRuns)
+      return asArray(this.parsedSplits?.Run.AttemptHistory.Attempt).filter(a => selectTime(a));
+    else
+      return asArray(this.parsedSplits?.Run.AttemptHistory.Attempt);
+  }
+
+  get runSliderMarks(): number[] {
+    const firstRunId = this.runAttempts[0]['@_id'];
+    const lastRunId = this.runAttempts[this.runAttempts.length - 1]['@_id'];
```

Comment:
I know this is my code but actually it's dumb ^^
We already computed lastRunId, we can just do
```suggestion
    const lastRunId = this.latestAttemptNumber;
```